common/static/coffee/src/discussion/discussion.coffee

```diff
@@ -20,7 +20,7 @@ if Backbone?
 
     setSortComparator: (sortBy) ->
       switch sortBy
-        when 'date' then @comparator = @sortByDateRecentFirst
+        when 'activity' then @comparator = @sortByDateRecentFirst
```

Comment:
`sortByDateRecentFirst` calls `pinnedThreadsSortComparatorWithDate` which looks at `created_at` value of thread. Is that what we want?